### DISCUSSION TITLE
Have Laravel Debugbar output from remote website pipe into Laravel Ray instead

### DISCUSSION THREAD

**Discussion by gkimpson** [spatie/laravel-ray#228] [Ideas]
I have Laravel Debugbar running locally but obviously this isn't a good thing to have running on a remote server - would it be possible to have Laravel Debugbar data from a live/remote site and have this pipe directly into my Laravel Ray instance?

**Comment by freekmurze** [comment]
That currently isn't possible.

**Reply by gkimpson** [reply]
thanks I didn't think it would be but was worth asking the question cheers :)
